src/legend/svgLegend.ts

```diff
@@ -357,35 +364,42 @@ export class SVGLegend implements ILegend {
 
         legendItems
             .select(SVGLegend.LegendIcon.selectorName)
-            .attr({
-                "cx": (d: LegendDataPoint, i) => d.glyphPosition.x,
-                "cy": (d: LegendDataPoint) => d.glyphPosition.y,
-                "r": iconRadius,
-            })
-            .style({
-                "fill": (d: LegendDataPoint) => {
+            .attr(
+                "cx", (d: LegendDataPoint, i) => d.glyphPosition.x
+            )
+            .attr(
+                "cy", (d: LegendDataPoint) => d.glyphPosition.y
+            )
+            .attr(
+                "r", iconRadius,
+            )
+            .style(
+                "fill", (d: LegendDataPoint) => {
                     if (hasSelection && !d.selected)
                         return LegendBehavior.dimmedLegendColor;
                     else
                         return d.color;
                 }
-            });
+            );
 
         legendItems
             .select("title")
             .text((d: LegendDataPoint) => d.tooltip);
 
         legendItems
             .select(SVGLegend.LegendText.selectorName)
-            .attr({
-                "x": (d: LegendDataPoint) => d.textPosition.x,
-                "y": (d: LegendDataPoint) => d.textPosition.y,
-            })
+            .attr(
+                "x", (d: LegendDataPoint) => d.textPosition.x)
```

Comment:
Since it's less than 120 symbol we should write this function call into a single line